AdyenNetworking/APIClient/APIClient.swift

```diff
@@ -66,7 +66,9 @@ public final class APIClient: APIClientProtocol {
 
         log(urlRequest: urlRequest, request: request)
         
-        requestCounter += 1
+        DispatchQueue.main.async {
+            self.requestCounter += 1
```

Comment:
I would feel safer if we weakify self 😁

Reply:
That won't change anything, the closure is executed in a timely manner so `self` won't be retained for that long anyway

Reply:
that's the assumption, I have the same assumption and its probably the case, but we will never be 100% sure, would we?, that's why I would be safer this way.

but its not a big issue :-).